CustomEntitySearchTest/LookupTests.cs

```diff
@@ -0,0 +1,191 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Linq;
+using AzureCognitiveSearch.PowerSkills.Common;
+/// <summary>
+/// This is my attempt at a unit test. I took information from VSE documentation and the team's current SkillsetsTest.cs
+/// </summary>
```

Comment:
unnecessary comment, can you just delete this?

The mentioned file is from another repository as well, so not useful to anyone reading it here